plugins/modules/ds8000_volume.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
```

Comment:
The code here is similar to other code we already reviewed, so I didn't add comments here.